client/components/domains/register-domain-step/index.jsx

```diff
@@ -71,77 +72,40 @@ function getQueryObject( props ) {
 	};
 }
 
-function processSearchStatQueue() {
```

Comment:
Yep, that's the idea. We have `railcar` now. It was introduced to get some more info on user search queries, which I've never seen us use.